### PR TITLE
7 av implement s3 storage

### LINKED ISSUE #7: implement s3 storage

### PR DESCRIPTION
**Pull request by AnnaVi11arrea1**
closes #7

### CODE REVIEW

File: app/controllers/application_controller.rb
Comment (viviancreates) [inline app/controllers/application_controller.rb:1]: Great job fixing devise issue

File: db/migrate/20241120022548_add_photo_to_events.rb
Comment (viviancreates) [inline db/migrate/20241120022548_add_photo_to_events.rb:1]: good update to schema!

**Review by viviancreates — APPROVED** [review]
Great attention to detail adding photo upload functionality

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <%= f.error_notification %>
 
   <div class="form-inputs">
-    <%= f.input :email, required: true, autofocus: true %>
+    <%= f.hidden_field :email, required: true, autofocus: true %>
```
Comment (hatcheta1) [inline app/views/devise/registrations/edit.html.erb:8]: nice job customizing a devise view template!

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -77,7 +90,7 @@ def set_event
 
   # Only allow a list of trusted parameters through.
   def event_params
-    params.require(:event).permit(:name, :application_due_at, :started_at, :information, :application_link, :tags, :address)
+    params.require(:event).permit(:photo, :name, :application_due_at, :started_at, :information, :application_link, :tags, :address)
```
Comment (hatcheta1) [inline app/controllers/events_controller.rb:93]: great job adding photo to your permitted parameters!

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -50,7 +64,7 @@ def set_user
   end
 
   # Only allow a list of trusted parameters through.
-  # def user_params
-  #   params.require(:user).permit(:email, :password, :password_confirmation, :username, :first_name, :last_name, :social_media, :about, :avatar)
-  # end
+  def user_params
```
Comment (hatcheta1) [inline app/controllers/users_controller.rb:67]: nice job uncommenting code before committing

**Review by hatcheta1 — COMMENTED** [review]
nice implementation of photo upload!

**Pull request merged**